### ISSUE TITLE
My PR was accepted still i dont see my name as part of the community ?

### ISSUE THREAD

**Issue by sadath-12** [EddieHubCommunity/LinkFree#5010]
### Description

My PR was accepted still i dont see my name as part of the community ?

**Comment by ChinmayMhatre** [comment]
Could you link the PR which was merged?

**Comment by SurajPratap10** [comment]
@sadath-12  if you are not able to find your profile on the website, then I can suggest you to:
- Must create a tag section and fill 3-4 tags according to your developer profile when u submit the data: "username" json file.
- Later, always search by typing 2 or 3 tags that u mentioned in your JSON file, these common tags may help to get you to see your profile. 

I am suggesting this because at first, I was also not able to see my profile when I typed frontend developer but when I typed 2-3 tags then my profile popped out with other peoples having those same tags.
